src/components/common/Button/index.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Story, Meta } from '@storybook/react/types-6-0'
+import { ButtonBase } from '.'
+import { DarkModeThemeToggler } from 'storybook/decorators'
+
+export default {
+  title: 'Common/Button',
+  component: ButtonBase,
+  decorators: [DarkModeThemeToggler],
+  argTypes: {
+    label: { control: 'text' },
+    theme: {
+      control: null,
+    },
+    as: {
+      control: null,
+    },
+    forwardedAs: {
+      control: null,
+    },
+  },
+} as Meta
+
+const Template: Story = (args) => <ButtonBase {...args}>{args.label}</ButtonBase>
```

Comment:
I would expect `Story<ButtonProps>` here, like in other stories

Reply:
For the fun of it if you are having trouble extracting props from StyledComponent, try inference
```ts
type PropsFromStyled<T extends StyledComponent<any,any>> = T extends StyledComponent<React.FC<infer U>, any> ? U : T

type ButtonProps = PropsFromStyled<typeof ButtonBase>
//or
type PropsFromFC<T extends React.FC> = T extends React.FC<infer U> ? U : T

Reply:
when i can't understand the types, i dont do inference :D